test/dotnet-new3.UnitTests/Sdk/DotnetCommand.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using dotnet_new3.UnitTests;
+using Microsoft.DotNet.Cli.Utils;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.TestFramework.Commands
+{
+    public class DotnetNewCommand : TestCommand
+    {
+        public DotnetNewCommand(ITestOutputHelper log, params string[] args) : base(log)
+        {
+            // Set dotnet-new3.dll as first Argument to be passed to "dotnet"
+            // And use full path since we want to execute in any working directory
+            Arguments.Add(Path.GetFullPath("dotnet-new3.dll"));
```

Comment:
is it a change comparing to SDK framework? Probably it would be good to keep all the new or changed files aside copied ones.

Reply:
Moved